scipy/sparse/csgraph/__init__.py

```diff
@@ -117,19 +176,19 @@
 Directed vs. Undirected
 -----------------------
 Matrices may represent either directed or undirected graphs.  This is
-specified throughout the csgraph module by a boolean keyword.  Graphs are
+specified throughout the ``csgraph`` module by a boolean keyword.  Graphs are
 assumed to be directed by default. In a directed graph, traversal from node
-i to node j can be accomplished over the edge G[i, j], but not the edge
-G[j, i].  In a non-directed graph, traversal from node i to node j can be
-accomplished over either G[i, j] or G[j, i].  If both edges are not null,
+``i`` to node ``j`` can be accomplished over the edge ``G[i, j]``, but not the edge
+``G[j, i]``.  In a non-directed graph, traversal from node ``i`` to node ``j`` can be
+accomplished over either ``G[i, j]`` or ``G[j, i]``.  If both edges are not null,
 and the two have unequal weights, then the smaller of the two is used.
 Note that a symmetric matrix will represent an undirected graph, regardless
-of whether the 'directed' keyword is set to True or False.  In this case,
+of whether the 'directed' keyword is set to ``True`` or ``False``.  In this case,
 using ``directed=True`` generally leads to more efficient computation.
 
-The routines in this module accept as input either scipy.sparse representations
-(csr, csc, or lil format), masked representations, or dense representations
-with non-edges indicated by zeros, infinities, and NaN entries.
+The routines in this module accept as input either `scipy.sparse <http://docs.scipy.org/doc/scipy/reference/sparse.html#module-scipy.sparse>`_
+representations (`csr`, `csc`, or `lil` format), masked representations,
```

Comment:
these backticks won't generate working links I think, because those names are not in the ``csgraph`` namespace. Needs to be prefixed with ``sparse.``